SIT.Manager.Avalonia/ViewModels/MainViewModel.cs

```diff
@@ -50,6 +51,7 @@ public partial class MainViewModel : ViewModelBase, IRecipient<PageNavigationMes
         ILogger<MainViewModel> logger,
         HttpClient httpClient)
     {
+        UpdateTranslationStrings((Languages)managerConfigService.Config.CurrentLanguageSelected);
```

Comment:
Accessing the localization system like this gives me the ick, see previous comments about services